pkg/kubelet/kubelet_pods.go

```diff
@@ -420,6 +420,15 @@ func truncatePodHostnameIfNeeded(podName, hostname string) (string, error) {
 	return truncated, nil
 }
 
+// GetUserNamespaceMappings returns the configuration for the sandbox user namespace
+func (kl *Kubelet) GetUserNamespaceMappings(pod *v1.Pod) (*runtimeapi.UserNamespace, error) {
+	return kl.usernsManager.GetUserNamespaceMappings(pod)
+}
+
+func (kl *Kubelet) getHostIDsForPod(pod *v1.Pod, containerUID, containerGID *int64) (hostUID, hostGID *int64, err error) {
+	return kl.usernsManager.getHostIDsForPod(pod, containerUID, containerGID)
+}
```

Comment:
How do we know that a signed int64 is OK to use here? Also, would it be better to use a 2-element struct for the UID and GID IDs?

Reply:
if the value overflows the 32bits value, then it won't be possible to map it to the user namespace and the error `fmt.Errorf("ID: %v not present in pod user namespace", id)` is returned.

Reply:
@sftim not sure I follow, a comment explaining that if an ID can't be mapped we return an error saying we didn't find a map? 

Can you please elaborate what you mean? Or a suggestion of what comment you would like?

Reply:
I'd add a comment about how we know that a signed int64 is the right size.

Reply:
I don't really see where it could fit and add value. Please re-open with a concrete suggestion if you see where it really helps.